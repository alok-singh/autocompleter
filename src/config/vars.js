// api debounce timein milliseconds
export const API_DEBOUNCE_TIME = 500;

// autocompleter api. can omit api key from here and put in environment variables
export const AUTOCOMPLETER_API_URL = 'http://www.omdbapi.com/?apikey=c4267e95&s=${KEYWORD}&type=movie';

// a random fallback image can be set according to brand
export const FALLBACK_IMAGE = 'http://www.omdbapi.com/src/poster.jpg';