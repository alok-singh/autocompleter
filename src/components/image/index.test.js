import React from 'react';
import { render } from '@testing-library/react';
import Image from './index';
import { API_DEBOUNCE_TIME, AUTOCOMPLETER_API_URL } from "../../config/vars";

test('unit tests for image components', () => {
  expect(render(<Image />).baseElement).toMatchSnapshot();
});

test('unit tests for image components with className component', () => {
  expect(render(<Image className="my-image" />).baseElement).toMatchSnapshot();
});