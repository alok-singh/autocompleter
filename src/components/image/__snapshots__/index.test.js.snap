// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`unit tests for image components 1`] = `
<body>
  <div>
    <img
      class="u-image "
    />
  </div>
</body>
`;

exports[`unit tests for image components with className component 1`] = `
<body>
  <div>
    <img
      class="u-image my-image"
    />
  </div>
</body>
`;
